src/io/flutter/inspector/FlutterStudioProfilersView.java

```diff
@@ -0,0 +1,406 @@
+    myBinder.bind(FlutterStudioMonitorStage.class, FlutterStudioMonitorStageView::new);
+    //myBinder.bind(CpuProfilerStage.class, CpuProfilerStageView::new);
+    //myBinder.bind(MemoryProfilerStage.class, MemoryProfilerStageView::new);
+    //myBinder.bind(NetworkProfilerStage.class, NetworkProfilerStageView::new);
+    //myBinder.bind(NullMonitorStage.class, NullMonitorStageView::new); // This is for no device detected
+    //myBinder.bind(EnergyProfilerStage.class, EnergyProfilerStageView::new);
+
+    myProfiler.addDependency(this)
+              .onChange(ProfilerAspect.STAGE, this::updateStageView);
+    updateStageView();
+  }
+
+  @Override
+  public void dispose() {
+  }
+
+  @VisibleForTesting
+  public <S extends FlutterStage, T extends FlutterStageView> void bind(@NotNull Class<S> clazz,
+                                                          @NotNull BiFunction<FlutterStudioProfilersView, S, T> constructor) {
+    myBinder.bind(clazz, constructor);
+  }
+
+  @VisibleForTesting
+  @NotNull
+  CommonButton getZoomInButton() {
+    return myZoomIn;
+  }
+
+  @VisibleForTesting
+  @NotNull
+  CommonButton getZoomOutButton() {
+    return myZoomOut;
+  }
+
+  @VisibleForTesting
+  @NotNull
+  CommonButton getResetZoomButton() {
+    return myResetZoom;
+  }
+
+  @VisibleForTesting
+  @NotNull
+  CommonButton getFrameSelectionButton() {
+    return myFrameSelection;
+  }
+
+  @VisibleForTesting
+  @NotNull
+  JToggleButton getGoLiveButton() {
+    return myGoLive;
+  }
+
+  @VisibleForTesting
+  public FlutterStageView getStageView() {
+    return myStageView;
+  }
+
+  @VisibleForTesting
+  @NotNull
+  SessionsView getSessionsView() {
+    return mySessionsView;
+  }
+
+  private void initializeStageUi() {
+    myToolbar = new JPanel(new BorderLayout());
+    JPanel leftToolbar = new JPanel(ProfilerLayout.createToolbarLayout());
+
+    myToolbar.setBorder(DEFAULT_BOTTOM_BORDER);
+    myToolbar.setPreferredSize(new Dimension(0, TOOLBAR_HEIGHT));
+
+    myCommonToolbar = new JPanel(ProfilerLayout.createToolbarLayout());
+    JButton button = new CommonButton(StudioIcons.Common.BACK_ARROW);
+    button.addActionListener(action -> {
+      myProfiler.setMonitoringStage();
+      //myProfiler.getIdeServices().getFeatureTracker().trackGoBack();
+    });
+    myCommonToolbar.add(button);
+    myCommonToolbar.add(new FlatSeparator());
+
+    JComboBox<Class<? extends FlutterStage>> stageCombo = new FlatComboBox<>();
+    JComboBoxView stages = new JComboBoxView<>(stageCombo, myProfiler, ProfilerAspect.STAGE,
+                                               myProfiler::getDirectStages,
+                                               myProfiler::getStageClass,
+                                               stage -> {
+                                                 // Track first, so current stage is sent with the event
+                                                 //myProfiler.getIdeServices().getFeatureTracker().trackSelectMonitor();
```

Comment:
Flutter had a way of tracking feature usage. Go ahead and use it instead if you want or remove the comment.